src/operator-docs/creation/empty.ts

```diff
@@ -1,6 +1,76 @@
 import { OperatorDoc } from '../operator.model';
 
 export const empty: OperatorDoc = {
-  'name': 'empty',
-  'operatorType': 'creation'
+  name: 'empty',
+  operatorType: 'creation',
+  signature: 'public empty<T>(scheduler?: IScheduler): Observable<T>',
+  parameters: [
+    {
+      name: 'scheduler',
+      type: 'IScheduler',
+      attribute: 'optional',
+      description:
+        'Allows scheduling the emission of the complete notification.'
+    }
+  ],
+  marbleUrl: 'http://reactivex.io/rxjs/img/empty.png',
+  shortDescription: {
+    description:
+      'Creates an Observable that emits no items to the Observer' +
+      ' and immediately emits a complete notification.',
+    extras: [
+      {
+        type: 'Tip',
+        text: `Just emits 'complete', and nothing else.`
```

Comment:
I know this comes from the rxjs 5 docs but I believe this could cause confusion. To me a beginner may think this actually emits the string `'complete'` and not understand the complete notification.

For instance, if I'm a new user and read this:

> Just emits 'complete', and nothing else.

 Then look above:

>Creates an Observable that emits no items

This could be frustrating. Is there a way we can enhance the wording here? Or maybe this one is better without the extra tip?

Reply:
I would remove the tip indeed @btroncone to avoid confusion as you mentioned.